sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/CHANGELOG.md

```diff
@@ -4,12 +4,25 @@
 
 ### Features Added
 
+* Added `Resource` to traces, logs, and metrics with default configuration.
+  ([#37837](https://github.com/Azure/azure-sdk-for-net/pull/37837))
+* Added resource detection for `Azure App Service` and `Azure Virtual Machine` environment. .
+  ([#37837](https://github.com/Azure/azure-sdk-for-net/pull/37837))
+
 ### Breaking Changes
 
 ### Bugs Fixed
 
 ### Other Changes
 
+* Update OpenTelemetry dependencies
+  ([#37837](https://github.com/Azure/azure-sdk-for-net/pull/37837))
+  - OpenTelemetry 1.5.1
+  - OpenTelemetry.Extensions.Hosting 1.5.1
+  - OpenTelemetry.Instrumentation.AspNetCore 1.5.1-beta.1
+  - OpenTelemetry.Instrumentation.Http 1.5.1-beta.1
+  - OpenTelemetry.Instrumentation.SqlClient 1.5.1-beta.1
```

Comment:
should also include a similar message in the Exporter Changelog

Reply:
Only for OpenTelemetry package?

Reply:
yup. that's what we've been doing :)